rules-reviewed/eap7/weblogic/weblogic-jms.windup.xml

```diff
@@ -109,13 +112,13 @@
                 <javaclass references="oracle.jms.AQjmsConnection" />
             </when>
             <perform>
-                <hint title="Oracle JMS Consumer" effort="1" category-id="mandatory">
+                <hint title="Oracle JMS consumers" effort="1" category-id="mandatory">
```

Comment:
I think this should be Oracle JMS connections, not consumers -- looks like a copy/paste gone wrong in the original. Whom do i ask about this?